roles/ocp_on_libvirt/README.md

```diff
@@ -1,3 +1,29 @@
+# OCP_on_libvirt role
+
+Role to setup a libvirt cluster ready to get provisioned with an OpenShift Container Platform release.
+
+## Parameters
+
+Name                                    | Required  | Default  | Description
+----------------------------------------|-----------|----------|-------------
+enable_conserver                        | No        | true     | Conserver is used to write the console output to a file for later monitoring. Disabling it may be required to monitor the VMs console during ongoing jobs.
+enable_redfish                          | No        | false    | By default, VMs are created with a Virtual Baseboard Management Controller (vBMC) that can be operated using the IPMI protocol. This parameter enables support for the Redfish protocol based on the sushy_tool Redfish Emulator.
+enable_virtualmedia                     | Only with enable_redfish=true | false | The Redfish protocol uses PXE as the default protocol to provide the hosts with the boot image. Currently we don't support the Redfish/PXE combination so, if you intend to use redfish in your setup, you must also enable virtualmedia.
+vbmc_user                               | Yes       | -        | User name to authenticate with the Redfish API.
+vbmc_pass                               | Yes       | -        | User password to authenticate with the Redfish API.
+externalMACAddress                      | Yes       | -        | MAC address to be assigned to the bootstrap VM, required for DHCP managed static addressing.
+bootstrapProvisioningIP                 | Yes       | -        | The IP address on the bootstrap VM where the provisioning services run while the installer is deploying the control plane (master) nodes
+rhsm_username                           | Yes       | -        | User name to log into the [Red Hat Access Portal](https://access.redhat.com). We strongly recommend to have the user name encrypted in an ansible-vault string.
+rhsm_password                           | Yes       | -        | User password to log into the [Red Hat Access Portal](https://access.redhat.com). We strongly recommend to have the user name encrypted in an ansible-vault string.
```

Comment:
Where are these used in the role?

Reply:
That's a good point.

The RHSM variables are not used in this role as it is.

The point is, for ocp_on_libvirt to create VMs supporting UEFI boot mode, the initial bootstrap image we server can not be the usual CentOS image, but RHEL. This also applies to the jumpbox host, which then needs to enable certain repositories so, for that, some RHSM variables are provided.

You can check the [bos2-ci-config PR](https://github.com/dci-labs/bos2-ci-config/pull/93/files) counterpart to this one.

Reply:
In any case, I'll remove those variables from the documentation.